### ISSUE TITLE
OutOfMemoryError

### ISSUE THREAD

**Issue by dulkith** [jhomlala/betterplayer#421]
I got this error
```
java.lang.OutOfMemoryError: 
  at dalvik.system.VMRuntime.newNonMovableArray (VMRuntime.java)
  at android.graphics.BitmapFactory.nativeDecodeStream (BitmapFactory.java)
  at android.graphics.BitmapFactory.decodeStreamInternal (BitmapFactory.java:863)
  at android.graphics.BitmapFactory.decodeStream (BitmapFactory.java:839)
  at android.graphics.BitmapFactory.decodeStream (BitmapFactory.java:877)
  at com.jhomlala.better_player.BetterPlayer.access$500 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.disposeRemoteNotifications (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.getBitmapFromExternalURL (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.access$000 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.access$002 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.access$100 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.access$400 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.access$602 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.buildMediaSource (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.dispose (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.isHTTP (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.lambda$setupPlayerNotification$0 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.onPictureInPictureStatusChanged (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.seekTo (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setAudioAttributes (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setDataSource (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setSpeed (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setTrackParameters (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setupMediaSession (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setupPlayerNotification (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer.setupVideoPlayer (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer$1.createCurrentContentIntent (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer$1.getCurrentLargeIcon (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer$1.lambda$getCurrentLargeIcon$1 (BetterPlayer.java)
  at com.jhomlala.better_player.BetterPlayer$1.lambda$null$0 (BetterPlayer.java)
  at com.jhomlala.better_player.-$$Lambda$BetterPlayer$1$uWg1n0P0FVg9Vcc8zRN9UXtmJkM.run (-.java)
  at java.lang.Thread.run (Thread.java:818)
```

**Comment by jhomlala** [comment]
@dulkith Please fill issue template correctly.

**Comment by jhomlala** [comment]
Fixed in 0.0.64.
